staff/carlos-luis-querecuto/airlines/index.js

```diff
@@ -0,0 +1,117 @@
+    }
+    return modal;
+}
+function admin(name,flights) {
+    var loop = true;
+    var op , todo, sorted;
+    var newflight = [
+        { id: 00, to: '', from: '', cost: 0, scale: false }
+    ]
+    while(loop){
+        op=prompt("Hi Admin "+name+" you can modify the DB with the following commands\n\n - 'create' : you cand add more flights to DB \n - 'delete' : you can errase flights from DB \n - 'exit : exit application ");
+        if(op==="create"){
+            if(flights.length <= 15){
+                newflight.id=flights.length;
+                newflight.to=prompt("Insert departure");
+                newflight.from=prompt("Insert destination");
+                newflight.cost=prompt("Insert price").toString();
+                newflight.scale=confirm("Have scales?");
+                flights.push(newflight);
+                alert("flight added Successfully! \n"+dataHandler(flights));
+            }else{
+                alert("Max quantity of Flights allowed are 15!")
+            }
+            
+        }
+        if(op==="delete"){
+            if(flights.length > 0){
+                todo=prompt("Insert ID of the Flight that will be erased");
+                flights=flights.filter(flight => (flight.id).toString() !== todo);
+                alert("flight erased Successfully!\n"+dataHandler(flights));
+            }
+            else{
+                alert("Flights list is Empty!");
+            }
+        }
+        if(op==="exit"){
+            alert("See ya admin "+name+"!");
+            return 0
+        }    
+    }
+}
+function user(name,flights){
+    op=prompt("Hi "+name+" before buy, you can see again our Flights sorted, please type one of the following commands\n\n - 'most': to see flightlist sorted from the most expensive\n - 'less': to see flightlist sorted from the cheapest\n - 'search' : to search flights with a especific price"); 
+    sorted=flights; 
+    if(op==="most"){
+        sorted.sort(function(a, b){return b.cost - a.cost});
+    }
+    if(op==="less"){
+        sorted.sort(function(a, b){return a.cost - b.cost});
+    }
+    if(op==="search"){
+        todo=prompt("Insert a price to search");
+        sorted=sorted.filter(flight => (flight.cost).toString() === todo);
+        if(sorted.length<=0){
+            alert("Sorry! we dont have results :(, here is our flights list");
+            sorted=flights;
+        }
+    }
+    var buy,find;
+    while(true){
+        buy=prompt(dataHandler(sorted)+"\n To buy, Please type ID of your desired flight");
+        find=sorted.filter(flight => (flight.id).toString() === buy);
+        if(find.length>0){
+            alert("Thank You for Buy!, Enjoy your Travel!");
+            return buy;
+        }
+        else{
+            alert("Sorry! we cant find a flight with that id :(");
+            find=sorted;
+        }
+    }
+}
+function skylabAirlines() {
+    var name, userBuy;
```

Comment:
```
var name,
      userBuy;

```
should be more polite.

Reply:
those variables are not assigned yet, maybe in only a line confussions could be avoided.